File: references/RndIndoorLaboratoryStocktaking.cs
using System;

namespace RnDLaboratoryStock.Core.Models;

public partial class RndIndoorLaboratoryStocktaking
{
    public int Id { get; set; }

    public int SessionId { get; set; }

    public string Cabinet { get; set; } = null!;

    public int Shelf { get; set; }

    public int EpcCount { get; set; }

    public DateTime? InsertedAt { get; set; }
}

File: app/src/main/java/com/example/rndlaboratorystock/Models/RndIndoorLaboratoryStocktaking.java
package com.example.rndlaboratorystock.Models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class RndIndoorLaboratoryStocktaking {
    @SerializedName("responseCode")
    public int ResponseCode;
    @SerializedName("data")
    public List<Data> Data = null;

    public class Data {
        @SerializedName("id")
        public int Id;
        @SerializedName("sessionId")
        public int SessionId;
        @SerializedName("cabinet")
        public String Cabinet;
        @SerializedName("shelf")
        public int Shelf;
        @SerializedName("epcCount")
        public int EpcCount;
        @SerializedName("insertedAt")
        public String InsertedAt;
    }
}
